build/build.ps1

```diff
@@ -6,24 +6,24 @@ $ErrorActionPreference = 'Stop'
 & "$PSScriptRoot/set-env.ps1"
 $all_ok = $True
 
-if ($Env:ENABLE_NATIVE -ne "false") {
-    $nativeSimulator = (Join-Path $PSScriptRoot "../src/Simulation/Native")
-    & "$nativeSimulator/build-native-simulator.ps1"
+if ($Env:ENABLE_QIRRUNTIME -ne "false") {
```

Comment:
I am curious, why is QIR runtime built before the native simulator now?

Reply:
We update QIR RT more frequently, and it is more likely to fail the build (than the native simulator). So if the build fails then it is more efficient to fail earlier than later. Some more info is in [Build the Most Failing Files First](https://github.com/kuzminrobin/code_review_notes/blob/master/misc_posts.md#build-the-most-failing-files-first).